mobile/scripts/ios/run.sh

```diff
@@ -31,4 +31,4 @@ fi
 echo "Installing app $APP on simulator $SIMULATOR_DEVICE_ID"
 open -a Simulator --args -CurrentDeviceUDID "$SIMULATOR_DEVICE_ID"
 xcrun simctl install "$SIMULATOR_DEVICE_ID" "$APP"
-xcrun simctl launch --console "$SIMULATOR_DEVICE_ID" im.status.Status-tablet
\ No newline at end of file
+xcrun simctl launch --console-pty "$SIMULATOR_DEVICE_ID" im.status.Status-tablet
```

Comment:
I hope it's not related to openssl, but for some reason when using `--console` with latest sdk the app is freezing. I'll do some tests with the old ssl as well

Reply:
Done.

Tested with openssl1 and it's the same. Probably something changed with the last Xcode/sdk update.